Cannot read properties of undefined (reading 'ssr')

I am getting an error when running `astro build` + `astro preview` and it seems is related to this package.

my astro config file:

```ts
import { defineConfig } from 'astro/config';
import mdx from '@astrojs/mdx';
import sitemap from '@astrojs/sitemap';
import biome from 'astro-biome';
import worker from '@astropub/worker';

// https://astro.build/config
export default defineConfig({
  site: 'https://example.com',
  integrations: [mdx(), sitemap(), worker(), biome()],
});
```
Error message:
```bash
16:23:50 [vite] ✓ 3 modules transformed.
16:23:50 [ERROR] [vite] x Build failed in 18ms
[at position 40] Cannot read properties of undefined (reading 'ssr')
file: ///IntroPhotoSection.astro?astro&type=script&index=0&lang.ts
  Stack trace:
    at Object.handler (///node_modules/.pnpm/@astropub+worker@0.1.0/node_modules/@astropub/worker/integration.js:20:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
*shortened some paths for privacy*

Any other information I might be able to provide:

packages:
```json
  "dependencies": {
    "@astrojs/check": "^0.5.6",
    "@astrojs/mdx": "^2.1.1",
    "@astrojs/rss": "^4.0.5",
    "@astrojs/sitemap": "^3.1.1",
    "@astropub/worker": "^0.1.0",
    "astro": "^4.4.5",
    "astro-biome": "^0.0.6",
    "typescript": "^5.3.3"
  },
  "devDependencies": {
    "@biomejs/biome": "1.5.3",
    "sass": "^1.71.1",
    "sharp": "^0.33.2",
    "typescript-esbuild": "^0.3.10"
  }
```

platform: Windows 11
node version: `v20.10.0`

--- a/packages/worker/integration.js
+++ b/packages/worker/integration.js
@@ -17,7 +17,7 @@ export default function AstroIntegration() {
 											const source = new MagicString(code)
 
 											// patch for globalThis.Worker
-											if (options.ssr) {
+											if (options && options.ssr) {
 												source.appendLeft(0, `import "@astropub/worker/polyfill";`)
 											}
 

--- a/packages/worker/integration.js
+++ b/packages/worker/integration.js
@@ -17,7 +17,7 @@ export default function AstroIntegration() {
 											const source = new MagicString(code)
 
 											// patch for globalThis.Worker
-											if (options && options.ssr) {
+											if (options?.ssr) {
 												source.appendLeft(0, `import "@astropub/worker/polyfill";`)
 											}
 

